nixos/modules/security/pam.nix

```diff
@@ -1199,12 +1201,27 @@ in
       description = lib.mdDoc "Message of the day shown to users when they log in.";
     };
 
+    users.motdFile = mkOption
+    {
```

Comment:
Could this be `mkOption {` so that it follows the convention of this file?

Reply:
I'll just run nixpkgs-fmt on this file, but this will result in a bigger commit (+712 -686).

Reply:
I just noticed, that nixpkgs-fmt does not fix this specific inconsistency. I've just implemented this small change.